R/collect.R

```diff
@@ -11,17 +11,26 @@
 #'  used to filter the predicted values before processing. This tibble should
 #'  only have columns for each tuning parameter identifier (e.g. `"my_param"`
 #'  if `tune("my_param")` was used).
+#' @param type One of `"long"` (the default) or `"wide"`. When `type = "long"`,
```

Comment:
I'm conflicted on the name and values for this argument. Very much open to suggestions. :)

Reply:
I think that a logical called `pivot_wider` is good. It implies that it is already long.

Reply:
Just to check, we decided that it's unlikely that we'll ever deviate from something binary? No other forms of "wide" metrics anticipated?